docs/end_to_end_tutorials/dev_practices/evaluation.md

```diff
@@ -0,0 +1,26 @@
+# Evaluation
+
+## Setting the Stage
+
+LlamaIndex is meant to connect your data to your LLM applications.
+
+Sometimes, even after diagnosing and fixing bugs by looking at traces, more fine-grained evaluation is required to systematically diagnose issues.
+
+LlamaIndex aims to provide those tools to make identifying issues and receiving useful diagnostic signals easy.
+
+Closely tied to evaluation are the concepts of experimentation and experiment tracking.
+
+While evaluating your pipeline, it is important to keep in mind what parameters your system may be sensitive to.
+
+## Aspects of Evaluation
```

Comment:
It might help to provide a short overview paragraph here or in the previous section to motivate what "e2e evaluation" and "component wise evaluation" are and why you need both components. The user needs to grasp our evaluation principles here before clicking into each section for more details